File: prime.c
#include<stdio.h>

int main(){
int n;
printf("enter number==>");
scanf("%d",&n);

if(n<2){
    printf("none");
}
else if(n==2){
    
    printf("prime");
}
else{
    int val;
    for(val=2;val<n;val+=1){
        if(n%val==0){
            printf("composite\n");
            break;
        }
    }
    if(val==n){
        printf("prime\n");

    }
        }

       printf("%s","Hello, World!\n");
       char s[100];
       fgets(s,100,stdin);
     


return 0; 
}

File: prime.py
a=int(input("enter number="))

if(a<2):
    print("none")
elif(a==2):
    print("prime")
else:
    for val in range(2,a):
        if a%val==0:
            print("composite")
            break
    else:
        print("prime")        
  
              
      
      
    